code/modules/mob/living/carbon/inventory.dm

```diff
@@ -137,20 +137,21 @@
 	if(!. || !I) //We don't want to set anything to null if the parent returned 0.
 		return
 
+	var/not_handled = FALSE
```

Comment:
what handled?

Reply:
if we actually unequipped an item, this is because we dont want to run this proc twice, once for carbons and once for humans

Reply:
im saying make a comment or name the var better!!

Reply:
oh ok